testing-guidance/workflow-tests/scripting/script-canvas-tests.md

```diff
@@ -60,54 +64,72 @@ Responsive Script Canvas editor with all 5 basic tools enabled. No Asset Process
+| Using View Menu                  | <ol><li>Zoom graph in and out.</li><li>Show selection:</li><ul><li>individual node</li><li>multiple connected nodes</li><li>group</li><li> notes</li><li>entire graph</li></ul></ul></li><li>Use "Go to" option to navigate to start and end of the chain (graph must have multiple nodes connected with each other).</li><li>Restore default layout of Script Canvas tool.</li><li>Use available keyboard shortcuts to actions mentioned above.</li></ol> | <ul><li>Show selection or show entire graph options are displaying graph accurately.</li><li>Default layout can be restored.</li> |
+| Using Tools Menu                  | <ol><li>Open and close tools available in Tools Menu.</li><li>Use available keyboard shortcuts to open and close all tools.<li>Perform resize, maximize, minimize, undock and dock actions with all of Script Canvas editor tools and drag them around the Script Canvas editor.</li></ol> | <ul><li>Tools are opened after selecting them in Tools Menu.</li><li>Shortcuts can be used to open/close tools.</li><li>The user is able to resize, minimize, maximize, undock, dock or drag Script Canvas editor tools around the Script Canvas editor.</li> |
+| Using Script Events [Preview] Menu          | <ul><li> This menu is soon to be deprecated and reworked, until its updated its ommited from testing </li><li> https://www.o3de.org/docs/user-guide/scripting/script-events/ can be followed for the current state of the functionality. |
+---
+
+### Area: Basic Script Canvas graph tests 
 Testing in this area verify basic functionality of script canvas graphs and more in-depth functionality of the default editor tools.
 
-### Project Requirements
-Any project with a new level. SC editor with the __SC Basic Editor Tools__ enabled
+**Project Requirements** 
+Any project with a new level. SC editor with the __SC Basic Editor Tools__ enabled.
 
-### References
-Default Script Canvas editor layout with the default tools enabled and highlighted
-![SC graph editor and default tools](images/SCeditor_default_layout.png)
+**References**
+Default Script Canvas editor layout with the default tools enabled and highlighted.
+[![SC graph editor and default tools](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/images/SCeditor_default_layout.png)](https://github.com/o3de/sig-content/blob/main/testing-guidance/workflow-tests/scripting/images/SCeditor_default_layout.png)
```

Comment:
Let's listify this so that the image renders offset and doesn't take away from the flow of the document.

You can also play with the formatting of this with [collapsed sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).